docs/experimental/ticket-auction.md

```diff
@@ -0,0 +1,402 @@
+Alice drops the price of her nft to 90 mutez.
+
+```sh
+$ tezos-client transfer 0 from alice to nft-auction \
+        --entrypoint "drop_price" \
+        --arg 90 --burn-cap 1
+
+        Transaction:
+              Amount: ꜩ0
+              From: tz1LrQB9HrJcUaD9NKEvV65tnaiU8trPXwmt
+              To: KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG
+              Entrypoint: drop_price
+              Parameter: 90
+              This transaction was successfully applied
+              Updated storage:
+                (Pair { 0x00000d4f0cf2fae2437f924120ef030f53abd4d4e520 ;
+                        90 ;
+                        10 ;
+                        True ;
+                        1611781430 ;
+                        600 }
+                      139)
+```
+
+### Purchase NFT
+
+Bob buys the nft by sending 90 mutez to the auction contract,
+calling the buy entrypoint, and sending the address of his wallet contract.
+The nft is sent to Bob’s wallet and Alice is sent the 90 mutez.
+
+```sh
+$ tezos-client transfer 0.00009 from bob to nft-auction \
+        --entrypoint "buy" \
+        --arg "\"$BOB_WALLET_ADDRESS%receive\"" \
+        --burn-cap 1
+
+        Transaction:
+          Amount: ꜩ0.00009
+          From: tz1bwfmSYqrhUTAoybGdhWBBefsbuhNdcC2Y
+          To: KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG
+          Entrypoint: buy
+          Parameter: "KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs%receive"
+          This transaction was successfully applied
+          Updated storage:
+            (Pair { 0x00000d4f0cf2fae2437f924120ef030f53abd4d4e520 ;
+                    90 ;
+                    10 ;
+                    False ;
+                    1611781430 ;
+                    600 }
+                  139)
+          Updated big_maps:
+            Unset map(139)[0]
+
+          Balance updates:
+            tz1bwfmSYqrhUTAoybGdhWBBefsbuhNdcC2Y ... -ꜩ0.00009
+            KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG ... +ꜩ0.00009
+            Internal operations:
+          Transaction:
+            Amount: ꜩ0.00009
+            From: KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG
+            To: tz1LrQB9HrJcUaD9NKEvV65tnaiU8trPXwmt
+            Balance updates:
+              KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG ... -ꜩ0.00009
+              tz1LrQB9HrJcUaD9NKEvV65tnaiU8trPXwmt ... +ꜩ0.00009
+
+        Internal operations:
+          Transaction:
+            Amount: ꜩ0
+            From: KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG
+            To: KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs
+            Entrypoint: receive
+            Parameter: (Pair 0x013d32a903fa4cf753f0d7075a7ab8395c501a2caf00 (Pair 1 1))
+
+            Updated storage:
+              { 0x0000b2d8083a660b2a77efe28a71bf09a933cd85613b ; 142 ; 2 ; 143 }
+            Updated big_maps:
+              Set map(142)[1] to (Pair 0x013d32a903fa4cf753f0d7075a7ab8395c501a2caf00 (Pair 1 1))
+
+```
+
+Now, it will be possible to see the purchased NFT at key 1
```

Comment:
Since most of this is done through the `tezos-client`, do you think it would be good to add the command to look in the `big_map`?

Reply:
added